setup.py

```diff
@@ -90,7 +90,9 @@ def setup_environmental_variables():
 
     with open("./web/.env", "r+") as file:
         file_data = file.read()
-        file_data = re.sub(r"NEXT_PUBLIC_SUPABASE_URL=.*", f"NEXT_PUBLIC_SUPABASE_URL={supabase_anon_key}", file_data)
+        # required for supabase middleware fix
+        file_data = re.sub(r"NEXT_PUBLIC_SUPABASE_ANON_KEY=.*", f"NEXT_PUBLIC_SUPABASE_ANON_KEY={supabase_anon_key}", file_data)
+        file_data = re.sub(r"NEXT_PUBLIC_SUPABASE_KEY=.*", f"NEXT_PUBLIC_SUPABASE_KEY={supabase_anon_key}", file_data)
```

Comment:
```suggestion
        file_data = re.sub(r"NEXT_PUBLIC_SUPABASE_KEY=.*", f"NEXT_PUBLIC_SUPABASE_KEY={supabase_anon_key}", file_data)
```
redundant, let's use the same variable for this